internal/testrunner/runners/system/tester.go

```diff
@@ -1528,12 +1528,10 @@ func (r *tester) validateTestScenario(ctx context.Context, result *testrunner.Re
 		logger.Warn("Validate mappings found (technical preview)")
 		exceptionFields := listExceptionFields(scenario.docs, fieldsValidator)
 
-		mappingsValidator, err := fields.CreateValidatorForMappings(r.dataStreamPath, r.esClient,
```

Comment:
As the process to load the fields from the package and ECS has been removed, these parameters are not needed anymore.

The schema to be used to validate fields with ECS is the one set (optionally) via the method `WithMappingValidatorFallbackSchema`